openpower/package/petitboot/petitboot.mk

```diff
@@ -19,6 +19,8 @@ PETITBOOT_CONF_OPTS += --with-ncurses --without-twin-x11 --without-twin-fbdev \
 	      HOST_PROG_SHUTDOWN=/usr/libexec/petitboot/bb-kexec-reboot \
 	      $(if $(BR2_PACKAGE_BUSYBOX),--with-tftp=busybox)
 
+PETITBOOT_AUTORECONF_ENV += PETITBOOT_VERSION=`cat $(PETITBOOT_VERSION_FILE) | cut -d '-' -f 2-`
```

Comment:
I must be missing why do you want to use a different version variable than ${PETITBOOT_VERSION} from above?

Seems like this could become a regularly repeated pattern.  Do we need to do it for other packages?